packages/evolution-generator/README.md

```
@@ -358,27 +358,30 @@ In your survey logic, conditionals play a key role in determining if the widget
+| Field                                        | Description                                                              | Type                        |
+| -------------------------------------------- | ------------------------------------------------------------------------ | --------------------------- |
+| conditionalName                              | Name of the conditional                                                  | string                      |
+| logicalOperator                              | Logical operator (optional)                                              | && or \|\|                  |
+| path                                         | Path to the response object for comparison                               | string                      |
+| [comparisonOperator](#comp)                  | Operator for comparison                                                  | ===, ==, >, <, >= or <=     |
+| value                                        | Value for the comparison                                                 | number or string            |
+| [parentheses](#par)                          | Parentheses (optional)                                                   | ( or )                      |
+| [hidden_value](#conditionals_hidden_value) | Value to use when the widget is hidden (conditional is false) (optional) | number or string or boolean |
 
 > <span id="comp">**Note:**</span> The `comparisonOperator` field helps compare respondent response with the specified value. It determines how the respondent's answer should be evaluated in the conditional logic. For example, `>=` signifies that the condition is true when path response is greater than or equal to the value.
 
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)